tla-io/src/main/scala/at/forsyte/apalache/io/lir/TlaType1TagPrinter.scala

```diff
@@ -7,6 +7,9 @@ class TlaType1TagPrinter extends TypeTagPrinter {
     tag match {
       case Untyped()           => "Untyped"
       case Typed(tt: TlaType1) => tt.toString
+      // IrGenerators generates type with arbitrary integers. In order not to crash JSON (en|de)coding tests,
+      // we need to not throw on this variant of Typed, but we erase the type with Untyped()
+      case Typed(_: Int) => "Untyped"
       // unexpected type, output as much as we can
       case Typed(_) => "Typed[%s](%s)".format(tag.getClass.getSimpleName, tag)
     }
```

Comment:
Why is this class called `TlaType1TagPrinter`, though it is printing arbitrary `Untyped()` as well? Shall we rename it to `TagPrinter`?

Reply:
I guess if we're sticking to the pattern, `DefaultTagPrinter`